src/main.sh

```diff
@@ -25,9 +25,13 @@ parseInputs () {
   export google_project="${INPUT_GOOGLE_PROJECT}"
   export DEV_PROJECT="${INPUT_GCR_GOOGLE_PROJECT}"
   export google_application_credentials=""
```

Comment:
If you define the google_application_credentials env var with a default as part of the action, why redefine it here?

Reply:
I think that's a good question - I wonder if it's making sure that it's an empty variable? @smark88 - do you know? 

At any rate- I think this would also be worth looking into in a follow up ticket.